org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/documentation/SonarLintDocumentation.java

```diff
@@ -38,6 +38,7 @@ public class SonarLintDocumentation {
   public static final String RULES = BASE_DOCS_URL + "/using-sonarlint/rules/";
   public static final String RULES_SELECTION = RULES + "#rule-selection";
   public static final String TROUBLESHOOTING_LINK = BASE_DOCS_URL + "/troubleshooting/";
+  public static final String ADVANCED_CONFIGURATION = BASE_DOCS_URL + "team-features/advanced-configuration/";
```

Comment:
I think it's missing a `/` at the beginning

Reply:
True, I fixed that